hpx/runtime/threads/policies/shared_priority_queue_scheduler.hpp

```diff
@@ -69,6 +69,8 @@ static_assert(false,
 #define SHARED_PRIORITY_SCHEDULER_LINUX
 #endif
 
+// #define SHARED_PRIORITY_SCHEDULER_DEBUG_NUMA
```

Comment:
Remove?

Reply:
it's like a flag that can be swtiched on/off, if you remove it, then anyone debugging might not know that it is useful...
I like to leave these in because this time next week, something else will break and I'll be back debugging it - it gets hard to maintain all the branches when you have to keep removing and putting back small snippets.